build-scripts/config_common.cmake

```diff
@@ -262,3 +262,7 @@ if (WAMR_BUILD_LOAD_CUSTOM_SECTION EQUAL 1)
     add_definitions (-DWASM_ENABLE_LOAD_CUSTOM_SECTION=1)
     message ("     Load custom section enabled")
 endif ()
+if (WAMR_BUILD_STACK_GUARD_SIZE GREATER 0)
+    add_definitions (-DWASM_STACK_GUARD_SIZE=${WAMR_BUILD_STACK_GUARD_SIZE})
+    message("     Custom stack guard size: " ${WAMR_BUILD_STACK_GUARD_SIZE})
```

Comment:
How about changing `message(` to `message (`?

Reply:
OK